src/components/buttons/button-group/ButtonGroup.vue

```diff
@@ -6,22 +6,60 @@ export interface Props {
   color?: ContextColorsType;
   variant?: 'default' | 'outlined' | 'text';
   size?: 'sm' | 'lg';
+  required?: boolean;
+  modelValue?: number | number[];
```

Comment:
hmm, at least in a few of the use-cases we have we use a custom value in buttons 

So we would need to be able to replicate this here too.

https://github.com/rotki/rotki/blob/83cadba74d99679c46c46df9b295c42f5b27fff0/frontend/app/src/components/staking/eth/EthValidatorFilter.vue#L30

Reply:
means we need to make it generic, and add `value` to the button component...